pkg/resources/schema.go

```diff
@@ -89,23 +91,24 @@ var schemaSchema = map[string]*schema.Schema{
 // Schema returns a pointer to the resource representing a schema.
 func Schema() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: CreateContextSchema,
-		ReadContext:   ReadContextSchema(true),
-		UpdateContext: UpdateContextSchema,
-		DeleteContext: DeleteContextSchema,
+		CreateContext: CommonCreateWrapper(resources.Schema, CreateContextSchema),
```

Comment:
let's add a check in one of the acceptance tests (or dedicated one) that after adding these, we see the tracking info in the query history. can be added in a separate PR

Reply:
Added in desc for as TODO in next pr